### DISCUSSION TITLE
Media information incorrectly displaying through Google Cast

### DISCUSSION THREAD

**Discussion by howesey2k** [music-assistant/support#2268] [Q&A] [answered]
Hi, I was wondering if anyone can help me or has the same problem. When I play any song through the YouTube or Spotify add on in music assistant the media artist and media album show as the media title. They show correctly in the music assistant add-on. This seems to be since I have changed over from beta to stable. I have shutdown my pi and tried reinstalling the beta but still have the same problem.
![Screenshot_20240512-204650](https://github.com/music-assistant/hass-music-assistant/assets/76223973/49c39895-5bff-42c4-a8e4-b681522f8a61)
![Screenshot_20240512-204629](https://github.com/music-assistant/hass-music-assistant/assets/76223973/c84112c7-eaa0-47c1-8427-f3b21047ee28)

**Comment by OzGav** [comment]
Have you cleared the browser cache and the MA cache (MA SETTINGS>>CORE>>CACHE CONTROLLER)

**Reply by howesey2k** [reply]
I have tried that thanks. No luck.

**Reply by howesey2k** [reply]
Just done a complete clean install on my raspberry pi 5 and having the same issue.

**Reply by OzGav** [reply]
OK I understand what you are rferring to now. Will covert to an issue and see if this is fixable.

**Reply by howesey2k** [reply]
Can I ask what has happened to this? It was showing up in the backlog yesterday as a bug an now it isn't there. Many thanks

**Reply by OzGav** [reply]
It’s in the next column. Don’t worry.

**Reply by howesey2k** [reply]
This appears in my log. Does this help?

2024-05-14 15:49:29.409 WARNING (MainThread) [music_assistant.providers.ytmusic] Skipping sync of item ytmusic://album/MPREb_or3kvAzrpYS - error details: Album is missing artist(s)

**Reply by howesey2k** [reply, marked as answer]
Thank you. This is now fixed after the update. I really appreciate it. Keep up the great work.
